notebooks/kubeflow_pipelines/cicd/labs/kfp_cicd.ipynb

```diff
@@ -247,8 +247,35 @@
    "cell_type": "markdown",
```

Comment:
<p>I think the workflow you propose here (edit cloudbuild.yaml in one repo clone + cp the cloudbuild.yaml in the other repo + push from that other repo) may be too much overhead, since you'll need to do that for every single change to the cloudbuild.yaml in the original clone (in the way the workflow is currently described) making the debugging quite cumbersome. </p><p> </p><p>What about asking the students (in text in a markdown cell) to edit directly the cloudbuild.yaml using the github editor on their fork? Then they can also create the tag/realease from the github UI, simplifying the debugging workflow very much.</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
That idea sounds simpler and great!
I will rewrite the instruction and let you know again.